config/environments/production.rb

```diff
@@ -24,7 +24,7 @@
   # config.assets.css_compressor = :sass
 
   # Do not fallback to assets pipeline if a precompiled asset is missed.
-  config.assets.compile = false
+  config.assets.compile = true
```

Comment:
Can you say a bit in the commit message about the motivation for this change at this point?

Should we also squash the two commits related to this, or are they distinct work?

Reply:
They're distinct. I was wondering about putting the second one in its own branch, but it is needed, we got an error in production on one of our OHSU assets without this setting (and found another error while testing this, which Jamie fixed, and that PR got merged earlier).

Reply:
Sounds good. Just a brief comment on each so we (and OHSU) have breadcrumbs later would be great.

As it stands, I have to admit that I don't understand why this was necessary at this point, but I bet it's straight-forward and you can explain it in a sentence or two. :dancing_men: